composer.json

```diff
@@ -91,7 +91,8 @@
         "phpunit/phpunit": "^5.6",
         "se/selenium-server-standalone": "^2.52",
         "stripe/stripe-php": "^4.1",
-        "symplify/easy-coding-standard": "^2.3"
+        "symplify/easy-coding-standard": "^2.4",
```

Comment:
I think you can drop this dependency, since it's already included in your custom coding standard: https://github.com/SyliusLabs/CodingStandard/blob/e50212b2be8f1e5984c69b9bdf205b32daa06688/composer.json#L15